backend/openapi.yaml

```diff
@@ -3298,13 +3302,225 @@ paths:
                 $ref: '#/components/schemas/SmsMessageTransactional'
         '400':
           description: Bad Request
-        "401":
+        '401':
           description: Unauthorized
-        "429":
+        '429':
           description: Too many requests. Please try again later.
-        "500":
+        '500':
           description: Internal Server Error
+  /transactional/sms:
+    get:
+      security:
+        - bearerAuth: []
+      tags:
+        - SMS
+      summary: 'List transactional SMSes'
+      parameters:
+        - in: query
+          name: limit
+          description: max number of messages returned
+          required: false
+          schema:
+            type: integer
+            minimum: 1
+            maximum: 100
+            default: 10
+        - in: query
+          name: offset
+          description: offset to begin returning messages from
+          required: false
+          schema:
+            type: integer
+            minimum: 0
+            default: 0
+        - in: query
+          name: status
+          description: status of messages to filter for
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+              enum: [UNSENT, ACCEPTED, SENT, DELIVERED, ERROR]
```

Comment:
how are we aligning these with [the statuses supported by Twilio](https://support.twilio.com/hc/en-us/articles/223134347-What-are-the-Possible-SMS-and-MMS-Message-Statuses-and-What-do-They-Mean-)? 

(not sure whether these are the same as the ones in the delivery reports, but don't think aligning with that is that important)